### DISCUSSION TITLE
Azure DevOps Git repo migration to Github self hosted along with pull requests in Azure DevOps

### DISCUSSION THREAD

**Discussion by tejaanand-OT** [community/community#118312] [Enterprise]
Hi Team,

Is there any process or tools available to migrate source code along with pull request history from Azure DevOps to GitHub Enterprise. Currently checked  **gh ado2gh** but it seems like useful only while migrating from Azure DevOps to Github Enterprise cloud.

Thank you.

**Comment by queenofcorgis** [comment]
Hi there, 

Do you have an account manager already for this? If not, I recommend contacting our [Sales team](https://github.com/enterprise/contact) to ensure you can have the smoothest possible migration.
